File: java/yapay-gateway-client/src/main/java/br/com/yapay/gateway/model/PassengerData.java
package br.com.yapay.gateway.model;

import com.google.gson.annotations.SerializedName;

public class PassengerData {

	@SerializedName("primeiroNome")
	private String firstName;

	@SerializedName("segundoNome")
	private String middleName;

	@SerializedName("sobrenome")
	private String lastName;

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getMiddleName() {
		return middleName;
	}

	public void setMiddleName(String middleName) {
		this.middleName = middleName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

}

File: net/YapayGatewayLib/model/PassengerData.cs
using System;
using Newtonsoft.Json;

namespace YapayGatewayLib.Model
{
    public class PassengerData
    {
    [JsonProperty(PropertyName = "primeiroNome")]
	public string firstName{get;set;}

    [JsonProperty(PropertyName = "segundoNome")]
	public string middleName{get;set;}

    [JsonProperty(PropertyName = "sobrenome")]
	public string lastName{get;set;}
    }
}
